_inc/client/security/jetpack-backup.jsx

```diff
@@ -0,0 +1,132 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+import { FEATURE_SITE_BACKUPS_JETPACK } from 'lib/plans/constants';
+
+export class JetpackBackup extends Component {
+	static propTypes = {
+		siteRawUrl: PropTypes.string,
+		rewindState: PropTypes.string,
+	};
+
+	static defaultProps = {
+		siteRawUrl: '',
+		rewindState: '',
+	};
+
+	getVaultPressContent = () => {
+		return (
+			<SettingsGroup
+				module={ { module: 'backups' } }
+				support={ {
+					text: __(
+						'Backs up your site to the global WordPress.com servers, ' +
+							'allowing you to restore your content in the event of an emergency or error.'
+					),
+					link: 'https://help.vaultpress.com/get-to-know/',
+				} }
+			>
+				{ __( 'Your site is backed up.' ) }
+			</SettingsGroup>
+		);
+	};
+
+	getRewindMessage() {
+		const { siteRawUrl, rewindStatus } = this.props;
+		const rewindState = get( rewindStatus, [ 'state' ], false );
+
+		switch ( rewindState ) {
+			case 'provisioning':
+				return {
+					title: __( 'Provisioning' ),
+					icon: 'info',
+					description: __( 'Jetpack Backup is being configured for your site.' ),
+					url: '',
+				};
+			case 'awaiting_credentials':
+				return {
+					title: __( 'Awaiting credentials' ),
+					icon: 'notice',
+					description: __(
+						'You need to enter your server credentials to finish configuring Jetpack Backup.'
+					),
+					url: 'https://wordpress.com/settings/security/' + siteRawUrl,
+				};
+			case 'active':
+				return {
+					title: __( 'Active' ),
+					icon: 'checkmark-circle',
+					description: __( 'Your site is being backed up.' ),
+					url: 'https://wordpress.com/activity-log/' + siteRawUrl,
+				};
+		}
+	}
+
+	getRewindBanner = () => {
+		const { title, icon, description, url } = this.getRewindMessage();
+
+		return (
+			<Banner
+				title={ title }
+				icon={ icon }
+				feature="rewind"
+				description={ description }
+				className="is-upgrade-premium jp-banner__no-border"
+				href={ url }
+			/>
+		);
+	};
+
+	render() {
+		const { rewindStatus, vaultPressData } = this.props;
+
+		const rewindState = get( rewindStatus, [ 'state' ], false );
```

Comment:
I'm seeing this same `get()` for a second time, should we have a helper for it? Or even better, since we use only the `state` from it, pass only the `state` as a prop? Or even better, have a dedicated selector that does this? 😉

Reply:
Agreed, I think it would be appropriate to fix this along with https://github.com/Automattic/jetpack/issues/14114